website/docs/cloud-docs/workspaces/health.mdx

```diff
@@ -60,6 +60,12 @@ Terraform Cloud pauses health assessments if the latest run ended in an errored
 
 Health assessments may take longer to complete when you enable health assessments in many workspaces at once or your workspace contains a complex configuration with many resources.
 
+### On Demand Assessments
+
+If you are an administrator for the workspace, you should see a **Start health assessment** button on the Health pages with Ready status if you have all your [workspace requirements](/terraform/cloud-docs/workspaces/health#workspace-requirements) are satisfied. 
+
+  ![On Demand Assessments Screenshot: Copy generated token](/img/docs/on-demand-assessments.png)
+
```

Comment:
Should we talk mention the helper text at all?

Also, we might want to add a blurb about expectations after clicking the button.
Something like.
"If the call to assess is successful, the assessment will begin. Based on factors like the size of the workspace and network latency, the assessment could take several minutes."

Reply:
By helper text you mean the toasts and error messages?

Good point about explaining the next steps.

Reply:
> By helper text you mean the toasts and error messages?

Yep. it. could be as simple as noting that there will be a box with messages about why you might not be able to kick off an on-demand assessment. It doesn't need to be in-depth, just a little something to make sure people know what to look for.

Reply:
what does "if the call to assess is successful"? in which cases would it not be? 

also - is there any throttling on how often a user can kick off one of these assessments?

Reply:
If the user is able to click the button and the assessments were successfully triggered on the backend, they will see a toast like this -
![success](https://user-images.githubusercontent.com/18754820/224110936-020be955-9666-421c-b476-345ca1653fdf.png)

However, due to some reason after the button click, we are not able to trigger assessments there will be an error toast shown to the users. The reasons for blocking are surfaced on the UI in the following call to workspace endpoint. 
<img width="1920" alt="error" src="https://user-images.githubusercontent.com/18754820/224110993-48e11559-7cfd-4184-8e66-5fdd299540a4.png">

There is no throttling added on the endpoint right now since we have limited this to a browser-only calls. And the "Start health assessment" button is disabled once you click the button until the assessment completes.

Reply:
we want to include only absolutely necessary screenshots to the documentation since it increases maintenance burden for every UI change. i've added some rephrases and questions that will make the documentation itself clear to the user.